docs/sources/panels/visualizations/graph-panel.md

```diff
@@ -28,9 +28,10 @@ Use these settings to refine your visualization.
 - **Line width -** The width of the line for a series. (default 1).
 - **Staircase -** Draws adjacent points as staircase.
 - **Area fill -** Amount of color fill for a series. (default 1, 0 is none)
-- **Fill gradient -** XXX
+- **Fill gradient -** Degree of gradient on the area fill. 0: no gradient, 10: a steep gradient.
```

Comment:
```suggestion
- **Fill gradient -** Degree of gradient on the area fill. (0 is no gradient, 10 is a steep gradient. Default is XX.
```

Reply:
Please enter the default value.

Reply:
Thanks for the feedback, will incorporate!